RapidReact/src/main/java/frc/robot/subsystem/ShooterSubsystem.java

```diff
@@ -1,13 +1,34 @@
 package frc.robot.subsystem;
 
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
 import frc.robot.config.Config;
 
 public class ShooterSubsystem extends BitBucketsSubsystem {
 
+  private TalonSRX doSomethingElseIdk;
+  private TalonSRX launchTheBalls;
```

Comment:
The shooter is actually using REV motors I think, and we should name them something that makes sense, like shooterMotor and feederMotor.